src/Core/L10n.php

```diff
@@ -436,7 +436,9 @@ public function getLanguageCodes(bool $international = false): array
 	{
 		$iso639 = new \Matriphe\ISO639\ISO639;
 
-		$languages = [];
+		// In ISO 639-2 undetermined languages have got the code "und".
+		// There is no official code for ISO 639-1, but "un" is not assigned to any language.   
+		$languages = ['un' => $this->t('Undetermined')];
```

Comment:
Other location for the constants I tentatively named `UNDETERMINED_LANGUAGE`.

Reply:
Done